src/Components/WelcomePage/WelcomePage.css

```diff
@@ -1,25 +1,26 @@
 body {
-  font-family: 'DYLOVASTUFF', sans-serif;
-  background-image: url('https://cdn.midjourney.com/3d47c815-3670-49a7-84bc-efe8d6b541da/0_3.webp');
+  font-family: 'Share-TechMono', sans-serif;
+  background-image: url('/Users/shane/mod4/capstone/front-end/src/assets/HyperLoom.png');
```

Comment:
@sdmisra We are receiving a CI error on this. Can you update the URL to correct file path in the app?

Reply:
[Fixed with this commit](https://github.com/The-Never-Ending-Story/front-end/pull/20/commits/0b7862d6e7b7acd28a00a9b751c5de50dd7600a5)

We can switch the Links to NavLinks if we want to style those elements differently in the future.